src/components/time/README.md

```diff
@@ -1,58 +0,0 @@
-### Examples
```

Comment:
should we keep the READMEs until styleguidist is removed?

if not, we'll need to remove from `https://github.com/box/box-ui-elements/blob/master/scripts/styleguide.config.js#L69`